assets/adaptivecard.json

```diff
@@ -17,7 +17,7 @@
                         "size": "Large",
                         "weight": "Bolder",
                         "color": "<% if (success) { %>Good<% } else { %>Attention<%} %>",
-                        "text": "nf-core/msa v${version} - ${runName}",
+                        "text": "nf-core/multiplesequencealign v${version} - ${runName}",
```

Comment:
```suggestion
                        "text": "nf-core/multiplesequencealign ${version} - ${runName}",
```

Reply:
`assets/slackreport.json` and `assets/email_template.html` don't have the `v`

Reply:
Ok - i will put your suggestion in another PR together with Julia's ones :)